cmake/PlasmaTargets.cmake

```diff
@@ -0,0 +1,84 @@
+include(BuildOptimizations)

```

Comment:
Since BuildOptimizations.cmake is now only 15 lines long, it might be preferable to just move all of its contents to this file.